_guides/analytics/deep_dive_analytics.md

```diff
@@ -0,0 +1,361 @@
+and sends the analytics data to the third-party provider,
+[Google Analytics](https://developers.google.com/analytics/devguides/collection/amp-analytics/):
+
+{% highlight html linenos %}
+<amp-analytics type="googleanalytics" config="https://example.com/analytics.account.config.json">
+<script type="application/json">
+{
+  "requests": {
+    "pageview": "https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}",
+    "event": "https://example.com/analytics?eid=${eventId}&elab=${eventLabel}&acct=${account}"
+  },
+  "vars": {
+    "account": "ABC123"
+  },
+  "triggers": {
+    "trackPageview": {
+      "on": "visible",
+      "request": "pageview"
+    },
+    "trackAnchorClicks": {
+      "on": "click",
+      "selector": "a",
+      "request": "event",
+      "vars": {
+        "eventId": "42",
+        "eventLabel": "clicked on a link"
+      }
+    }
+  },
+  'transport': {
+    'beacon': false,
+    'xhrpost': false,
+    'image': true
+  }
+}
+</script>
+</amp-analytics>
+{% endhighlight %}
+
+## Where to send analytics data: type attribute
+
+AMP is designed to support two common patterns of data collection:
+
+* Ingestion by a publisher-owned endpoint for in-house analytics systems.
+* Ingestion by a vendor-owned endpoint for interoperability with a vendor solution
+(for example, [Google Analytics](https://developers.google.com/analytics/devguides/collection/amp-analytics/),
+[Chartbeat](http://support.chartbeat.com/docs/),
+[Adobe Analytics](https://helpx.adobe.com/marketing-cloud/analytics.html)).
+
+To send analytics data to an analytics provider, include the type attribute in the
+`amp-analytics tag` and set its value to the appropriate vendor,
+as defind in the [amp-analytics specification](/docs/reference/extended/amp-analytics.html).  
+
+For example: `<amp-analytics type="googleanalytics">` sends analytics data
+to the third-party analytics provider, Google Analytics.
+To send data to publisher-owned endpoint, simply don’t include the type attribute;
+the analytics data is sent to the defined endpoints for each
+[request](/docs/guides/analytics/deep_dive_analytics.html#what-data-gets-sent-requests-attribute).
+
+Analytics vendor configurations are a quick way
+to get started with `amp-analytics`.
+You should consult your vendor’s documentation and
+help resources for further guidance.
+As previously mentioned,
+the list of vendors who’ve already integrated with AMP, as well as links
+to their specific documentation can be found in the
+[amp-analytics specification](/docs/reference/extended/amp-analytics.html). 
+
+If you’re an analytics vendor,
+learn more about
+[integrating your own analytics configuration into AMP HTML](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/integrating-analytics.md).
+
+## Load remote configuration: config attribute
+
+You don’t have to include all `amp-analytics` configurations in your AMP pages.
```

Comment:
You don't have to include all of the configuration for `amp-analytics` entirely on your AMP page.